starter-agents/crypto-portfolio-tracker/app.py

```
@@ -0,0 +1,162 @@
+
+CRYPTO_API = "https://agent-gateway-kappa.vercel.app/v1/crypto-feeds/api/prices"
+
+
+def get_crypto_prices():
+    """Fetch real-time crypto prices from free API (no auth required)."""
+    try:
+        resp = requests.get(CRYPTO_API, timeout=10)
+        resp.raise_for_status()
+        data = resp.json()
+        return data.get("prices", {})
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)